llvm/lib/Transforms/InstCombine/InstCombineAndOrXor.cpp

```diff
@@ -101,16 +101,16 @@ Value *InstCombinerImpl::insertRangeTest(Value *V, const APInt &Lo,
 ///    (icmp eq (A & B), A) equals (icmp ne (A & B), 0)
 ///    (icmp ne (A & B), A) equals (icmp eq (A & B), 0)
 enum MaskedICmpType {
-  AMask_AllOnes           =     1,
-  AMask_NotAllOnes        =     2,
-  BMask_AllOnes           =     4,
-  BMask_NotAllOnes        =     8,
-  Mask_AllZeros           =    16,
-  Mask_NotAllZeros        =    32,
-  AMask_Mixed             =    64,
-  AMask_NotMixed          =   128,
-  BMask_Mixed             =   256,
-  BMask_NotMixed          =   512
+  AMask_AllOnes = 1,
+  AMask_NotAllOnes = 2,
+  BMask_AllOnes = 4,
+  BMask_NotAllOnes = 8,
+  Mask_AllZeros = 16,
+  Mask_NotAllZeros = 32,
+  AMask_Mixed = 64,
+  AMask_NotMixed = 128,
+  BMask_Mixed = 256,
+  BMask_NotMixed = 512
```

Comment:
This is unrelated.

Reply:
Yes it from the formatter, i notice them but don't want to push and restart CI, once end i will update it and solve any other comments if there are new